_posts/2022-10-27-kani-bolero.md

```diff
@@ -0,0 +1,317 @@
+---
```

Comment:
Can you rename to `2022-10-27-using-kani-with-the-bolero-property-testing-framework`? This name will be part of the link when the post is generated.

Reply:
Done